monkey/infection_monkey/dropper.py

```diff
@@ -43,18 +47,31 @@ def __init__(self, args):
         arg_parser.add_argument('-l', '--location')
         self.monkey_args = args[1:]
         self.opts, _ = arg_parser.parse_known_args(args)
-
+        self._default_server = None
+        self._default_server_port = None
         self._config = {'source_path': os.path.abspath(sys.argv[0]),
                         'destination_path': self.opts.location}
 
     def initialize(self):
         LOG.debug("Dropper is running with config:\n%s", pprint.pformat(self._config))
+        self._default_server = self.opts.server
+        try:
+            self._default_server_port = self._default_server.split(':')[1]
+        except KeyError:
+            self._default_server_port = ''
+
+        if self._default_server:
+            if self._default_server not in WormConfiguration.current_server:
+                LOG.debug("Added current server: %s" % self._default_server)
+                WormConfiguration.current_server = self._default_server
+            else:
+                LOG.debug("Default server: %s is already in command servers list" % self._default_server)
```

Comment:
Instead of just copy-pasting export this code to a function and use it in both files.

Reply:
DM if you want me to elaborate.